### ISSUE TITLE
TODO

### ISSUE THREAD

**Issue by hosthans** [hosthans/domain_generalization#7]
- Pretrained/Randomized weights
- erweiterer Header (mit z.B: Dropout Layer)
- Learning Rate Schedules
- Early Stopping
- Cross Validation für Training
- Optimizer verändern 
- Batch-Size verändern
- evtl. anderes Modell
